implementations/rust/ockam/ockam_command/src/reset/static/long_about.txt

```diff
@@ -0,0 +1,16 @@
+-----------------------------------------------------------------------
+WARNING:
+
+The `reset` command is dangerous and should be used with caution as it will remove all local
+state PERMANENTLY. This behaviour can be useful for cleaning up your development environment
+and starting afresh from scratch. However, you'll typically want to use specific delete commands such as
+`node delete`, `identity delete`, and `space delete` for more granular control on deletes.
+
+-----------------------------------------------------------------------
```

Comment:
Note that this must look good both in the `--help` command output and in the [docs page](https://docs.ockam.io/reference/command).

We could try removing the `----` lines and also removing any line break, otherwise it will look broken in the command.